src/doc.rs

```diff
@@ -0,0 +1,354 @@
+use super::ast::{Arg, ArgNames, Statement, TypeAst, TypedModule};
+use itertools::Itertools;
+use std::collections::HashMap;
+#[derive(Debug, PartialEq)]
+struct RawComment {
+  pub line_no: usize,
+  pub content: String,
+}
+
+#[derive(Debug, PartialEq)]
+pub struct DocBlockManager {
+  pub(self) current_block: Vec<RawComment>,
+  pub(self) blocks: Vec<Vec<RawComment>>,
+  num_comments: usize,
```

Comment:
Is this used for anything? I couldn't spot it

Reply:
Look in parser.rs, it's what the comment lines get added to.

Reply:
I mean specifically the num_comments property

Reply:
Ah, it's used in the comment_count method that's module protected.  It's meant primarily for testing purposes.